### DISCUSSION TITLE
403 오류

### DISCUSSION THREAD

**Discussion by danpung2** [fsoftwareengineer/todo-application#78] [질문과 답변] [answered]
진행하면서 오류가 없었는데 5장 마지막에 테스트하면서 로그인해보니 계속 403이 뜨면서 메인페이지로 이동을 못하고 계속 로그인 페이지로 리디렉트 됩니다 프론트 문제인가요 백 문제인가요?

**Comment by fsoftwareengineer** [comment, marked as answer]
프론트화면에서 개발자 도구를 켠 후 해당 에러를 다시 실행해보고 네트워크탭을 캡쳐해 올려주시기바랍니다.
프론트엔드에서 토큰 저장을 제대로 하지 않았거나 백엔드에서 토큰 복호화가 제대로 안되는 것일 수 있습니다. 더불어 스프링 프로젝트에서 cors가 제대로 설정되어있는지 확인 해 주시기 바랍니다.

**Reply by danpung2** [reply]
토큰 저장이 제대로 안되고 있었습니다. 감사합니다.
